app/parse.py

```diff
@@ -29,8 +29,6 @@ class Product:
 
 PRODUCT_FIELD = [field.name for field in fields(Product)]
 
-_driver: WebDriver | None = None
-
 
 def get_driver() -> WebDriver:
     return _driver
```

Comment:
Now you code not work. Did you test it? 
You does not need function get_driver and set_driver just pass driver into another function

Reply:
Yes, I tested. All tests succeeded and got all tables. Please try again.

Reply:
You must delete get_driver and set_driver functions and don't use them in code. If you need WebDriver in some functions, WebDriver instance must be passed as argument to them